knip.jsonc

```diff
@@ -1,6 +1,7 @@
 {
 	"$schema": "https://unpkg.com/knip@next/schema.json",
 	"entry": ["src/index.ts!", "script/setup*.js"],
+	"ignoreBinaries": ["cat", "gh"],
```

Comment:
fyi @navin-moorthy - this fixes the `lint:knip` CI failure. https://github.com/webpro/knip/issues/116 & https://github.com/webpro/knip/issues/117 have a bit more context.